esphome/components/online_image/online_image.cpp

```diff
@@ -120,9 +124,14 @@ void OnlineImage::update() {
 
 #ifdef USE_ONLINE_IMAGE_PNG_SUPPORT
   if (this->format_ == ImageFormat::PNG) {
-    this->decoder_ = esphome::make_unique<PngDecoder>(this);
-  }
+    this->decoder_ = make_unique<PngDecoder>(this);
+  } else
 #endif  // ONLINE_IMAGE_PNG_SUPPORT
+#ifdef USE_ONLINE_IMAGE_BMP_SUPPORT
+      if (this->format_ == ImageFormat::BMP) {
```

Comment:
```suggestion
  if (this->format_ == ImageFormat::BMP) {
```
If the else is removed, no need to indent this further

Reply:
That was the formatter =)